sky/shell/platform/mojo/platform_view_mojo.cc

```diff
@@ -57,6 +57,9 @@ PlatformViewMojo::~PlatformViewMojo() {
 void PlatformViewMojo::Init(mojo::Shell* shell) {
   mojo::ConnectToService(shell, "mojo:native_viewport_service", &viewport_);
 
+  // Grab the application connector so that we can connect to services later
+  shell->CreateApplicationConnector(GetProxy(&connector_));
```

Comment:
The code in the file is oddly inconsistent here. I can make it be mojo::GetProxy everywhere.